boot/bootdata/bootcdregtest/regtest.cmd

```diff
@@ -11,14 +11,14 @@ if "%errorlevel%"=="1" (
     exit 0
 )
 
-move C:\ReactOS\bin\redirtest1.dll C:\ReactOS\bin\kernel32test_versioned.dll
-move C:\ReactOS\bin\testdata\redirtest2.dll C:\ReactOS\bin\testdata\kernel32test_versioned.dll
-if exist "C:\ReactOS\bin\AHKAppTests.cmd" (
+move %WINDIR%\bin\redirtest1.dll %WINDIR%\bin\kernel32test_versioned.dll
```

Comment:
Be careful about the environment variables syntax when being used within a batch script.